src/foremast/consts.py

```diff
@@ -140,6 +142,54 @@ def find_config():
     return configurations
 
 
+def _remove_empty_entries(entries):
+    """Remove empty entries in a list"""
+    valid_entries = []
+    for entry in entries:
```

Comment:
Minor optimization would be `set(entries)`. Not a real concern, but something to note.